tests/std/tests/VSO_0157762_feature_test_macros/test.compile.pass.cpp

```diff
@@ -714,6 +714,20 @@ STATIC_ASSERT(__cpp_lib_execution == 201603L);
 #endif
 #endif
 
+#if _HAS_CXX23

```

Comment:
Should we skip this for EDG?
```suggestion
#if _HAS_CXX23 && !defined(__EDG__) // TRANSITION, EDG concepts support
```

Reply:
I went with `defined(__cpp_lib_concepts)` as that is the condition in yvals_core